frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelWelcome.tsx

```diff
@@ -0,0 +1,63 @@
+import { IconInfo, IconNight, IconNotebook, IconSearch } from '@posthog/icons'
+import { LemonButton } from '@posthog/lemon-ui'
+import { useActions } from 'kea'
+
+import { KeyboardShortcut } from '../../components/KeyboardShortcut'
+import { sidePanelStateLogic } from '../sidePanelStateLogic'
+
+export const SidePanelWelcome = (): JSX.Element => {
+    const { closeSidePanel } = useActions(sidePanelStateLogic)
+
+    return (
+        <div className="flex flex-col items-center justify-center m-4 my-10 flex-1">
+            <h1 className="font-bold text-4xl text-center">
+                Welcome to <span className="text-primary-3000">PostHog&nbsp;3000</span>
+            </h1>
+
+            <p className="max-w-120 text-center">
+                <b>PostHog 3000</b> is the codename for our revamped user experience. It's currently in beta, and we'd
+                love to hear your feedback!
+            </p>
+            <ul className="space-y-4">
```

Comment:
One other major thing is "🔵 New Nav/Top Bar"

Reply:
I guess it is 🤔 but then also not really :D I think that falls under the "general UI refresh". But I also think Cory is going to change all my words so waiting on his thoughts :D